server/views/pages/bookVisit/checkVisitDetails.njk

```diff
@@ -53,7 +41,7 @@
           actions: {
             items: [
               {
-                href: "book-visit/select-visitors",
+                href: "/book-visit/select-visitors",
```

Comment:
The `href`s in this template can now use the path constants - e.g. here `paths.BOOK_VISIT.SELECT_VISITORS`